bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/retriever/AbstractCategoriesRetriever.java

```diff
@@ -66,6 +66,12 @@ public abstract class AbstractCategoriesRetriever extends AbstractRetriever {
      */
     protected List<String> identifiers;
 
+    /**
+     * Filter type that should be used when fetched. usually uid but we can define it explicitly in the implementation
+     * specific and should be checked in subclass implementations.
+     */
+    protected String filterType;
```

Comment:
Here also categoryIdType is a better name and is consistent with the setter method bellow.